backbone/src/main/java/org/researchstack/backbone/answerformat/PasswordAnswerFormat.java

```diff
@@ -0,0 +1,28 @@
+package org.researchstack.backbone.answerformat;
+
+import android.text.InputType;
+
+/**
+ * Created by TheMDP on 1/4/17.
+ */
+
+public class PasswordAnswerFormat extends TextAnswerFormat {
+
+    static final int DEFAULT_PASSWORD_MIN_LENGTH    = 4;
```

Comment:
don't think they should be private, would want a single source of "min" and "max" that can be referenced